twitter-subset-selection/src/Individual.cpp

```diff
@@ -61,4 +62,15 @@ void Individual::mutate() {
     (*this)[index] = !at((unsigned int)index);
 }
 
+bool Individual::paredoDominates(const Individual &opponent) const {
+    return (this->accuracy > opponent.accuracy) && (this->numFeaturesActive < opponent.numFeaturesActive);
```

Comment:
This is not correct: both objective functions don't have to be better. As long as one is greater and the other is greater than or equal to.

Reply:
oh shit this code is easier

Reply:
wait, don't we need at least one strictly greater (or less) than?  The code you propose lets both these values be equal to one another.

Reply:
True but I think the negation of that statement is enough to check for dominance. For example, consider the following: this = (95, 10), opponent = (92, 8); in this case this does not dominate opponent, right (this's accuracy is greater(which we want to maximize) but num of features is greater(which we want to minimize))? And just making sure that accuracy is not less and num of features is not greater yields the same result: this does not dominate opponent. 

Another example in which This does dominate Opponent: This = (95, 10), Opponent = (95, 12). Again checking the negation yields the same result: !(this.accuracy < opponent.accuracy) = true !(this.numFeatures > opponent.numFeatures) = true.

Can you think of a scenario the negation won't work?

Reply:
You got me there.